python/pynessie/_log.py

```diff
@@ -31,22 +29,8 @@ def show_log(
 
     def generator() -> Generator[CommitMeta, Any, None]:
         for i in raw_log:
-            if committer and i.committer != committer:
```

Comment:
note that this isn't currently tested in the existing `test_nessie_cli.py` tests because the `author` field can't be set when committing things and therefore is always empty. Once we have https://github.com/projectnessie/nessie/issues/1326 we can adjust the tests and make sure we can filter based on the `author` field